### PR TITLE
Upgrade to Ruby 2.6 Alpine

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: github-pages/Dockerfile [hunk 1]
```diff
@@ -1,24 +1,20 @@
 # https://pages.github.com/versions/
-FROM ruby:2.5-slim
+FROM ruby:2.6-alpine
```
Comment (tianon) [inline github-pages/Dockerfile]: Given that https://pages.github.com/versions/ still lists Ruby 2.5, I'd prefer to stick to 2.5 here as well, but moving to Alpine seems sane as long as we pin it while we're here:

```suggestion
FROM ruby:2.5-alpine3.10
```

Comment (J0WI) [reply github-pages/Dockerfile]: See https://github.com/github/pages-gem/pull/647

Comment (J0WI) [reply github-pages/Dockerfile]: https://rubygems.org/gems/github-pages just mention `>= 2.3.0`

Comment (tianon) [reply github-pages/Dockerfile]: Yeah, that's reasonable, but my goal with this image was to be able to test my GitHub pages deployments locally, and for that IMO it makes more sense to mirror their production deployment more than what the Gem technically supports. :+1:

**Pull request merged**